app/server/app/public/data/data.json

```diff
@@ -0,0 +1,115 @@
+{
+  "intro": "<em>How’s My Waterway</em> pulls data from multiple databases at EPA and other federal agencies through web services. Below is a summary of data included in <em>How’s My Waterway</em>.",
+  "content": [
+    {
+      "description": "ATTAINS contains information on water quality assessments, impaired waters, and total maximum daily loads (TMDLs), through data submitted by states under Clean Water Act sections <span data-glossary-term data-term='303(d) listed impaired waters (Category 5)'>303(d)</span> and 305(b).",
+      "extraContent": "<h2 style='display:block;margin-bottom:0.25rem;font-size:1.125em;line-height:1.125;font-family:Source Sans Pro,Helvetica Neue,Helvetica,Roboto,Arial,sans-serif;font-weight:bold;padding-bottom:0'>Impairment Category Filtering Tool:</h2><p><a href='attains' target='_blank' rel='noopener noreferrer'>How ATTAINS data are grouped in How’s My Waterway</a></p>",
```

Comment:
Production has more padding above `Impairment Category Filtering Tool:`. Can you add more padding there, so it matches production?
![image](https://user-images.githubusercontent.com/46329268/215820053-d89dcabd-419d-4f30-8ecd-4e3e774be50d.png)